packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -553,7 +553,7 @@ export default class Meeting extends StatelessWebexPlugin {
   namespace = MEETINGS;
   annotationInfo: AnnotationInfo;
   allowMediaInLobby: boolean;
-  sendSlots = new Map<MediaType, SendSlot>();
+  sendSlots: SendSlotsManager = new SendSlotsManager(LoggerProxy);
```

Comment:
Maybe `sendSlots` should be `sendSlotManager`, to keep it consistent with the `receiveSlotManager` property above.